ar/include/ikos/ar/format/text.hpp

```diff
@@ -85,6 +85,9 @@ class TextFormatter : public Formatter {
   /// \brief Format a global variable into text format
   void format(std::ostream&, GlobalVariable*) const;
 
+  // \brief Format just the head part of a function.
+  void format_header(std::ostream& o, const Function* f, Namer& namer) const;
```

Comment:
For consistency, remove the name of parameters
```suggestion
  void format_header(std::ostream&, const Function*, Namer&) const;
```

Reply:
Done in next commit.